src/lib.rs

```diff
@@ -70,6 +70,24 @@ impl Installer {
 
         Ok(solc_path)
     }
+
+    /// Extracts the solc archive at the version specified destination and returns the path to the
+    /// installed solc binary.
+    #[cfg(target_os = "windows")]
```

Comment:
You might have forgotten to constrain the target architecture?
```suggestion
    #[cfg(all(target_os = "windows", target_arch = "x86_64"))]
```